### PR TITLE
Readme update, download LTS version, lic section

### PR DESCRIPTION
**Pull request by dautovri**
Several changes:
- Added Download LTS section
- Added Licence section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -468,4 +472,8 @@ You may be wondering why do we have a container-based architecture for a Xen-cen
 
 In addition to that, while we plan to build a fully disagregated system (with even device drivers running in their separate domains) right now we are just getting started and having containers as a first step towards full disagreagation seems like a very convenient stepping stone.
 
-Let us know what you think by filing GitHub issues, and feel free to send us pull requests if something doesn't quite work.
+Let us know what you think by filing GitHub [issues](https://github.com/lf-edge/eve/issues/new/choose), and feel free to send us pull requests if something doesn't quite work.
+
+## Licence
```
Comment (rene) [inline README.md:477]: Licence -> License

File: README.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ EVE supports both ARM and Intel architectures and requires hardware-assisted vir
 
 To get its job done, EVE leverages a lot of great open source projects: [Xen Project](https://xenproject.org/), [Linuxkit](https://github.com/linuxkit/linuxkit) and [Alpine Linux](https://alpinelinux.org/) just to name a few. All of that functionality is being orchestrated by the Go microservices available under [pkg/pillar](pkg/pillar). Why pillar? Well, because pillar is the kind of a monolith we need to break out into true, individual microservices under [pkg/](pkg/).
 
+## Download EVE
+
+EVE LTS: [Donwload latest LTS](https://github.com/lf-edge/eve/releases/latest)
```
Comment (rene) [inline README.md:17]: Fix the typo pointed by Yetus

File: README.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ EVE supports both ARM and Intel architectures and requires hardware-assisted vir
...
+
```
Comment (rene) [inline README.md:18]: Doesn't make sense to also put a link for all releases (non-LTS)? I mean, a link to https://github.com/lf-edge/eve/releases page

Comment (dautovri) [reply README.md:18]: Yes, make sence, but we using pre-relase. 
For Pre-relase github dont apply semantic version sorting. Its means old but recently patched version can be latest in that case. Plus, link as **latest** for release section can be only one.

Comment (rene) [reply README.md:18]: ok, got it.

**Comment by rene** [comment]
LGTM

**Review by rene — APPROVED** [review]

**Review by christoph-zededa — APPROVED** [review]

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**